src/nvim/mark.c

```diff
@@ -887,6 +887,18 @@ void ex_changes(exarg_T *eap)
  */
 void mark_adjust(linenr_T line1, linenr_T line2, long amount, long amount_after)
 {
+  mark_adjust_internal(line1, line2, amount, amount_after, true);
+}
+
+void mark_adjust_nofold(linenr_T line1, linenr_T line2, long amount,
+    long amount_after)
+{
+  mark_adjust_internal(line1, line2, amount, amount_after, false);
+}
+
+static void mark_adjust_internal(linenr_T line1, linenr_T line2, long amount,
+    long amount_after, bool move_folds)
```

Comment:
Ah -- I get it now -- you mean the argument `move_folds` should be `adjust_folds`.